src/components/CommentBox/index.tsx

```diff
@@ -0,0 +1,21 @@
+import * as S from "./styles";
+
+type CommentBoxProps = {
+  user: string;
+  userImg: string;
+  children: React.ReactNode;
+};
+
+const CommentBox = ({ children, user, userImg }: CommentBoxProps) => {
+  return (
+    <S.Wrapper>
+      <S.CommentWrapper>{children}</S.CommentWrapper>
+      <S.UserInfo>
+        <img src={userImg} alt={`${user} avatar`} />
+        <span>{user}</span>
+      </S.UserInfo>
+    </S.Wrapper>
```

Comment:
Sempre que for trabalhar com comentários de pessoas, tente usar blockquote/quote/cite...

Aqui um artigo:

https://css-tricks.com/quoting-in-html-quotations-citations-and-blockquotes/